Makefile.include

```diff
@@ -10,6 +10,13 @@ RIOTCPU := $(abspath $(RIOTCPU))
 RIOTBOARD ?= $(RIOTBASE)/boards
 RIOTBOARD := $(abspath $(RIOTBOARD))
 
+# Top level git root of the project being built
+RIOTPROJECT ?= $(shell git rev-parse --show-toplevel)
+RIOTPROJECT := $(abspath $(RIOTPROJECT))
```

Comment:
Right. Do you have a suggestion for some kind of fall back behavior for
handling both cases transparently?
On Feb 21, 2015 4:25 PM, "Ludwig Ortmann" <notifications@github.com> wrote:

> In Makefile.include
> <https://github.com/RIOT-OS/RIOT/pull/2392#discussion_r25122802>:
>
> > @@ -10,6 +10,13 @@ RIOTCPU := $(abspath $(RIOTCPU))
> >  RIOTBOARD ?= $(RIOTBASE)/boards
> >  RIOTBOARD := $(abspath $(RIOTBOARD))
> >
> > +# Top level git root of the project being built
> > +RIOTPROJECT ?= $(shell git rev-parse --show-toplevel)
> > +RIOTPROJECT := $(abspath $(RIOTPROJECT))
>
> We ran into problems because some weird people actually chose to use our
> "release" tarball.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RIOT-OS/RIOT/pull/2392/files#r25122802>.
>